semantic_router/layer.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         self,
         routes: list[Route] = [],
         encoder_type: str = "openai",
-        encoder_name: str | None = None,
+        encoder_name: str | None = "text-embedding-ada-002",
```

Comment:
if I do `LayerConfig(encoder_type="cohere")` this change would mean that the OpenAI encoder still gets initialized by default — although not perfect I think it is best to keep default as `None` for now